scalardb-analytics-spark-sample/docker/Dockerfile.spark

```
@@ -0,0 +1,19 @@
+FROM openjdk:8u322-jre-slim
```

```suggestion
FROM eclipse-temurin:17-jre-jammy
```
Recently, we use eclipse temurin in the new project. So, I think it would be better to use it here.

---

Thank you for your suggestion! I changed the base image in https://github.com/scalar-labs/scalardb-samples/pull/58/commits/851bf800c6d2c801084686048f7edc6586afdef6.